src/playback/config.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable */
+
+import { selectFrom } from "../vendor/vectors";
+
+const SIZES = [
+  {size: "2160p60"},
+  {size: "2160p30"},
+  {size: "1440p60"},
+  {size: "1440p30"},
+  {size: "1080p60"},
+  {size: "1080p30"},
+  {size: "720p60"},
+  {size: "720p30"},
+  {size: "480p30"},
+  {size: "360p30"},
+  {size: "240p30"},
+  {size: "144p30"},
+  {size: "144p15"},
+];
+
+const ENCODINGS = [
+  {encoding: "VP9"},
+  {encoding: "H264"},
+];
+
+const SPEEDS = [
+  {speedLabel: "2", speed: 2},
+  {speedLabel: "1.5", speed: 1.5},
+  {speedLabel: "1.25", speed: 1.25},
+  {speedLabel: "1", speed: 1},
+  {speedLabel: "0.5", speed: 0.5},
+  {speedLabel: "0.25", speed: 0.25},
+];
+
+const TESTS = selectFrom(ENCODINGS)
+  .leftJoin('dummy', SIZES, 'dummy')
+  .leftJoin('dummy', SPEEDS, 'dummy')
+  .map(({encoding, size, speed, speedLabel}) => {
+    // H264.1080p30@1.25X_%_dropped_frames
+    const fullName1 =  encoding + '.' + size + '@' + speedLabel + 'X_%_dropped_frames';
+    const fullName2 = "PlaybackPerf." + fullName1;
+
+    return {
+      encoding,
+      size,
+      speed,
+      filter: {"eq": {"test": [fullName1, fullName2]}}
+    }
+  })
+  .toArray();
```

Comment:
Here we do a Cartesian join of the dimensions, and construct the `filter` that will pull the combination out of perfherder.  Notice that the there has been a change-in-test-name already.